examples/online-publications.json

```diff
@@ -669,6 +669,84 @@
                 "id": "promotion",
                 "name": "Aktionsmix"
             },
+            "teaser": {
+                "title": [
+                    {
+                        "locale": "de",
+                        "value": "20% auf alle italienische Rotweine"
+                    },
+                    {
+                        "locale": "fr",
+                        "value": "20% de rebais sur tous le vins itailens"
+                    },
+                    {
+                        "locale": "it",
+                        "value": "20% su tutti i vini italiani"
+                    }
+                ],
+                "description": [
+                    {
+                        "locale": "de",
+                        "value": "Auch auf betehende Aktionspreise"
+                    },
+                    {
+                        "locale": "fr",
+                        "value": "..."
+                    },
+                    {
+                        "locale": "it",
+                        "value": "..."
+                    }
+                ],
+                "stopper": [
+                    {
+                        "locale": "de",
+                        "value": "20%"
+                    },
+                    {
+                        "locale": "fr",
+                        "value": "20%"
+                    },
+                    {
+                        "locale": "it",
+                        "value": "20%"
+                    }
+                ],
+                "sales_price": "7.–",
+                "instead_price": [
+                    {
+                        "locale": "de",
+                        "value": "statt 8.–"
+                    },
+                    {
+                        "locale": "fr",
+                        "value": "au lieu de 8.–"
+                    },
+                    {
+                        "locale": "it",
+                        "value": "invece di 8.–"
+                    }
+                ],
+                "links": [
+                    {
+                        "locale": "de",
+                        "url": "https://www.denner.ch/de/shop/wein/weine.html"
+                    },
+                    {
+                        "locale": "fr",
+                        "url": "https://www.denner.ch/fr/shop/vin/vins.html"
+                    },
+                    {
+                        "locale": "it",
+                        "url": "https://www.denner.ch/it/shop/vino/vini.html"
+                    }
+                ],
+                "packshot": {
+                    "id": "104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1",
+                    "cdn_url": "https://denner.imgix.net/assets/104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1/web/051683_6.png"
+                },
```

Comment:
Changed to 
` "packshot_url":  "https://denner.imgix.net/assets/104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1/web/051683_6.png"`